src/Commands/DeployCommand.php

```diff
@@ -104,6 +104,7 @@ protected function buildProject(array $project)
         $this->call('build', [
             'environment' => $this->argument('environment'),
             '--asset-url' => $this->assetDomain($project).'/'.$uuid,
+            '--manifest' => Path::manifest(),
```

Comment:
This is the only command subcall that is project specific (the others are team specific)